config/spring19_seminar/colapair_bertF.conf

```diff
@@ -0,0 +1,60 @@
+// Direct probe model on cola-pair dataset, the parameters are frozen, we only use MaskLM to do acceptability judgement 
+// F stands for frozen
```

Comment:
Might as well spell this out in the filename.

Reply:
ok